### DISCUSSION TITLE
Livewire V3 File Upload Issue

### DISCUSSION THREAD

**Discussion by ali-hasnain-dev** [livewire/livewire#8820] [1. Help]
Hi everyone,

Hope you are doing well. I'm facing issue while using WithFileUploads functionality it is not making temporary url. In attached files you can see the code and issue.

Thanks.

![1](https://github.com/user-attachments/assets/db456b73-30f5-4afb-a45b-672f921ed1e2)
![2](https://github.com/user-attachments/assets/daa9e8c9-554e-4b47-a900-49e5c787f900)
![3](https://github.com/user-attachments/assets/25ff2353-aed6-4cf6-a863-b78166b6949a)

**Comment by danie-ramdhani** [comment]
try put the input file to the form

**Reply by ali-hasnain-dev** [reply]
Still facing same issue
![4](https://github.com/user-attachments/assets/de9780dd-0715-4897-a260-a166f408b772)
![5](https://github.com/user-attachments/assets/a5dd1114-11b5-4976-b1b9-d60ac0b83e84)
